front/app/components/SignUpIn/SignUpInModal.tsx

```diff
@@ -109,6 +113,10 @@ const SignUpInModal = memo<Props>(({ className, onMounted }) => {
     setMetaData(undefined);
   }, [metaData, authUser]);
 
+  if (!metaData?.inModal) {
+    return null;
+  }
```

Comment:
I'm guessing this code is needed to prevent the modal from poping up when the user is on a page where we show the SignUpIn component inline? If that's the case it might perhaps be better to figure out why the modal gets called instead of trying to remedy the situation here.

Reply:
I think this doesn't actually do anything but I'll double check and remove if thats the case